### PR TITLE
create alluvial fans page

### PR DESCRIPTION
**Pull request by rkelson**
this pull request will create a new page, add an image and update contacts.yaml
This page will be a place holder until I receive additional page content at which point a link to the page will be created on the Geoscience page

### CODE REVIEW

**Comment by steveoh** [comment]
[This commit](https://github.com/agrc/backseat-driver/pull/21/files) should contain your download ids if you wanted to update them.

**Comment by steveoh** [comment]
Don't forget to add a link on the https://5c780f59de491400095a1af1--gis-utah-gov.netlify.com/data/geoscience/ overview page as well in the stewardship spreadsheet so it shows up in the sgid index.

File: data/geoscience/alluvial-fans/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+      <p>Active alluvial fan landforms are defined by location, composition, and flooding characteristics. They are located at a topographic break between a mountain front and the piedmont, they are composed of unconsolidated alluvium, and exhibit flooding characteristics described previously. Inactive alluvial fan landforms are geologically older and are no longer subject to active fan flooding processes. As they age, inactive alluvial fans develop their own internal, tributary drainage networks that exhibit riverine flooding characteristics.</p>
+
+      <p>It is recommended that the alluvial fan landform GIS layer and supporting data be used by regulatory officials (federal, state, local, and tribal) when evaluating development and flooding hazards in their jurisdictions.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://gis.utah.gov/downloads/Active Alluvial Fan Landform Inventory - Wasatch and Oquirrh Ranges FINAL.zip">Active Alluvial Fan Landform Inventory Report- Wasatch and Oquirrh Ranges</a>
```
Comment (steveoh) [inline data/geoscience/alluvial-fans/index.html]: please do not use gis.utah.gov links. add the {{ style links. Also, I would recommend renaming this file to

1. Not have `FINAL` in the name
1. Remove the spaces in the file name.

**Comment by rkelson** [comment]
so I can use the MD style links in the html page?

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by steveoh** [comment]
you _can_ but you _shouldn't_. Here is the [example](https://github.com/agrc/gis.utah.gov#link-to-page) of how to do it in both markdown _and_ html.

File: data/geoscience/alluvial-fans/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+      <p>Active alluvial fan landforms are defined by location, composition, and flooding characteristics. They are located at a topographic break between a mountain front and the piedmont, they are composed of unconsolidated alluvium, and exhibit flooding characteristics described previously. Inactive alluvial fan landforms are geologically older and are no longer subject to active fan flooding processes. As they age, inactive alluvial fans develop their own internal, tributary drainage networks that exhibit riverine flooding characteristics.</p>
+
+      <p>It is recommended that the alluvial fan landform GIS layer and supporting data be used by regulatory officials (federal, state, local, and tribal) when evaluating development and flooding hazards in their jurisdictions.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="{{"https://drive.google.com/open?id=18J-3_GPHHGFo_TGosRuiPGWU0ktCq4AD"}}">Active Alluvial Fan Landform Inventory Report - Wasatch and Oquirrh Ranges</a>
```
Comment (steveoh) [inline data/geoscience/alluvial-fans/index.html]: external links do not require {{ }}

**Comment by rkelson** [comment]
this page(s) should be ready to merge. Once it is live the data steward can make changes

**Comment by steveoh** [comment]
remove the WIP

**Pull request merged**